jenkins/scripts/coverage/generate-index-html.py

```diff
@@ -6,9 +6,9 @@
 with open('out/index.csv') as index:
   index_csv = filter(lambda line: line, index.read().split('\n'))
 
+# noqa
```

Comment:
Is this comment incomplete?

Reply:
LGTM: This flake8 linter directive will silence the linter as discussed at 
http://flake8.pycqa.org/en/3.7.7/user/violations.html#in-line-ignoring-errors